platform.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2021-present PlatformIO <contact@platformio.org>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+
+from platform import system
+
+from platformio.managers.platform import PlatformBase
+from platformio.util import get_systype
+
+
+class Ststm8Platform(PlatformBase):
+
+    def configure_default_packages(self, variables, targets):
+        return PlatformBase.configure_default_packages(self, variables,
```

Comment:
Is there a reason for overriding the `configure_default_packages` method with an empty implementation?

Reply:
Hm, seems to be a relict from when I started my work based on the STM32 code (https://github.com/platformio/platform-ststm32/blob/develop/platform.py) but didn't need to modify anything after all. I'll remove it.